### PR TITLE
Adding troubleshooting section for DNS resolution when connected to VPN

### PR DESCRIPTION
**Pull request by sunilmut**

### CODE REVIEW

**Comment by bitcrazed** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[commandline/WSL/troubleshooting.md](https://github.com/sunilmut/CommandLine-Documentation/blob/live/commandline/WSL/troubleshooting.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C25%5C44a4c0e9-f7fc-ae3e-a431-dcb870cfc899%5CPullRequest%5C201707250105048609-111%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: commandline/WSL/troubleshooting.md [hunk 1]
```diff
@@ -16,6 +16,26 @@ ms.assetid: 6753f1b2-200e-49cc-93a5-4323e1117246
   This is the first release of this brand new technology and it is branded “beta” deliberately – it’s not yet complete! We know there are issues and incomplete features, you should expect some things to work and for many things to fail. But we appreciate you playing with this feature and helping us identify the issues we need to fix in order to deliver a great experience.
 
 
+### Bash loses network connectivity once connected to a VPN
+
+If after connecting to a VPN on Windows, bash loses network connectivity, try this workaround from within bash. This workaround will allow you to manually override the DNS resolution through `/etc/resolv.conf`.
```
Comment (benhillis) [inline commandline/WSL/troubleshooting.md]: Can we rephrase this to talk about Windows Subsystem for Linux instead of "bash"?

Comment (benhillis) [inline commandline/WSL/troubleshooting.md]: Oh I guess the rest of the document still hasn't been changed over, ignore.
